azclishell/app.py

```diff
@@ -402,6 +402,9 @@ def _special_cases(self, text, cmd, outside):
                 os.path.join(
                     SHELL_CONFIGURATION.get_config_dir(),
                     SHELL_CONFIGURATION.get_history())
+        if '--version' in text:
+            cmd = 'az ' + cmd
+            outside = True
```

Comment:
Yeah, but calling the method to do it does a system exit, and copy and pasting the same code wouldn't work if things changed in the cli